lib/msal-node/src/config/Configuration.ts

```diff
@@ -122,7 +122,7 @@ const DEFAULT_LOGGER_OPTIONS: LoggerOptions = {
 
 const DEFAULT_SYSTEM_OPTIONS: Required<NodeSystemOptions> = {
     loggerOptions: DEFAULT_LOGGER_OPTIONS,
-    networkClient: NetworkUtils.getNetworkClient(),
+    networkClient: new HttpClient(),
```

Comment:
I thought the initial idea was to ensure generic client setup, where `setNetworkClient()` would enable them to set the specific network client.